app/forms/provider_interface/interview_wizard.rb

```diff
@@ -5,7 +5,8 @@ class InterviewWizard
 
     VALID_TIME_FORMAT = /^(1[0-2]|0?[1-9])([:.\s]([0-5][0-9]))?([AaPp][Mm])$/.freeze
 
-    attr_accessor :time, :location, :additional_details, :provider_id, :application_choice, :provider_user, :current_step
+    attr_accessor :time, :location, :additional_details, :provider_id, :application_choice, :provider_user,
+                  :current_step, :path_history, :wizard_path_history, :action, :referer
```

Comment:
What's the difference between wizard_path_history and just path_history? Could we name these to be clearer?

Reply:
`wizard path history` is the name of the service holding the path information, and `path history` the actual path history (also reflecting the internal variable name of WizardPathHistory). We need `wizard_path_history` for access within the model and `path_history` so that the actual path history is stored on the wizard. Sticking to these names will make it easier to document their use in the upcoming wizard documentation, but if you can think of better names let me know. The PR is likely to sit until eob today.